pynag/Plugins/__init__.py

```diff
@@ -31,7 +31,7 @@
 import signal
 from optparse import OptionParser, OptionGroup
 from pynag.Utils import PerfData, reconsile_threshold
-from pynag.Parsers import extra_opts
+from pynag.Parsers import ExtraOptsParser
```

Comment:
I think we should be importing the symbol from where it is actually defined instead of transitively
```suggestion
from pynag.Parsers.extra_opts import ExtraOptsParser
```

Reply:
Here is a new release tag
https://github.com/pynag/pynag/releases/tag/pynag-1.1.2-1

Reply:
And here is the Pip https://pypi.org/project/pynag/